crates/test_utils/tests/lib.rs

```diff
@@ -0,0 +1,23 @@
+#![allow(
+    clippy::expect_used,
+    clippy::panic,
+    clippy::unwrap_in_result,
+    clippy::unwrap_used
+)]
```

Comment:
If you're adding it here, can you remove this annotation from the files corresponding to each of the submodules?